src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ListView.xaml

```diff
@@ -33,6 +33,8 @@
 			within the safe zone. Important to support devices with round corners or notches (iPhone X, Essential Phone) -->
 		<Setter Property="toolkit:VisibleBoundsPadding.PaddingMask"
 				Value="All" />
+		<Setter Property="Background"
```

Comment:
I usually don't set a background to listviews, is it really needed in this case?